__tests__/components/userDropDown.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { UserDropdown } from '@/components/index';
+import { userEvent, render, screen, waitFor } from '../test-utils';
+
+describe('UserDropdown', () => {
+  describe('when rendered', () => {
+    it('should render a profile drop down', () => {
+      render(<UserDropdown />);
+
+      expect(
+        screen.getByRole('button', { name: 'Profile options' })
+      ).toBeInTheDocument();
+    });
+  });
+
+  describe('when hovering', () => {
+    it('should show the log out menu and update accessibility', async () => {
```

Comment:
Change name to `should show the log out menu when hovering over down arrow and update accessibility`